oarepo_requests/ui/views.py

```diff
@@ -1,3 +1,21 @@
-from flask import Blueprint
+from oarepo_ui.resources.config import RecordsUIResourceConfig
+from oarepo_ui.resources.resource import RecordsUIResource
 
-blueprint = Blueprint("oarepo_requests_ui", __name__, template_folder="templates")
+class RequestUIResourceConfig(RecordsUIResourceConfig):
```

Comment:
Isn't there a suitable ui resource config specifically for requests? Provided by Invenio requests?